### PR TITLE
Reference blog post in README

### PR DESCRIPTION
**Pull request by sandermak**

### CODE REVIEW

**Review by Stephan202 — COMMENTED** [review]
Good one!

**Comment by Stephan202** [comment]
Suggested commit message:
```
Reference blog post in README (#282)
```

File: README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ focussing on maintainability, consistency and avoidance of common pitfalls.
 > Error Prone is a static analysis tool for Java that catches common
 > programming mistakes at compile-time.
 
+Read more on how Picnic uses Error Prone (Support) in the blog post [_Picnic
+loves Error Prone: producing high-quality and consistent Java
+code_][picnic-blog-ep-post].
```
Comment (sandermak) [inline README.md:20]: Replaying my internal debate with the opposite outcome 🙃. But it is at least 5 times more clickable, so let's go for it :)

**Review by japborst — APPROVED** [review]
Nice!

**Review by Stephan202 — APPROVED** [review]
:rocket:

**Pull request merged**